apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-steps/plan-step/index.tsx

```diff
@@ -12,32 +12,23 @@ import LaunchStep, { Props as LaunchStepProps } from '../../launch-step';
 import PlansGridFSE from '../../../../editor-plans-grid/src/plans-grid-fse';
 import './styles.scss';
 
-const PlanStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep, onNextStep } ) => {
-	const handleBack = () => {
-		onPrevStep?.();
-	};
-
-	return (
-		<LaunchStep className="nux-launch-plan-step">
-			<div className="nux-launch-step__header">
-				<div>
-					<Title>{ __( 'Choose a plan', 'full-site-editing' ) }</Title>
-					<SubTitle>
-						{ __(
-							'Pick a plan that’s right for you. Switch plans as your needs change. There’s no risk, you can cancel for a full refund within 30 days.',
-							'full-site-editing'
-						) }
-					</SubTitle>
-				</div>
-				<ActionButtons>
-					<BackButton onClick={ handleBack } />
-				</ActionButtons>
-			</div>
-			<div className="nux-launch-step__body">
-				<PlansGridFSE onSelect={ onNextStep } />
+const PlanStep: React.FunctionComponent< LaunchStepProps > = ( { onNextStep } ) => (
+	<LaunchStep className="nux-launch-plan-step">
+		<div className="nux-launch-step__header">
+			<div>
+				<Title>{ __( 'Choose a plan', 'full-site-editing' ) }</Title>
+				<SubTitle>
+					{ __(
+						'Pick a plan that’s right for you. There’s no risk, you can cancel for a full refund within 30 days.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Pick a plan that’s right for you. There’s no risk, you can cancel for a full refund within 30 days.' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/gutenboarding-launch-site-name-step)